api/services/administration/general/responses/getPolicyAction.yaml

```diff
@@ -0,0 +1,74 @@
+description: Get actions for access policies Response
+content:
+  application/json:
+    schema:
+      allOf:
+        - allOf:
```

Comment:
schema should inherit from `"../../../../../commons/schemas/responseItemWrapper.yaml"`.
only example should appear here.

you can take an example from `services/administration/general/responses/userGroup/getUserGroups.yaml`

Reply:
done